jdbc/src/main/java/io/kestra/jdbc/JdbcWorkerJobQueueService.java

```diff
@@ -0,0 +1,109 @@
+package io.kestra.jdbc;
+
+import io.kestra.core.exceptions.DeserializationException;
+import io.kestra.core.queues.QueueFactoryInterface;
+import io.kestra.core.queues.QueueInterface;
+import io.kestra.core.repositories.WorkerInstanceRepositoryInterface;
+import io.kestra.core.runners.*;
+import io.kestra.core.utils.Either;
+import io.kestra.jdbc.repository.AbstractJdbcWorkerJobRunningRepository;
+import io.kestra.jdbc.runner.JdbcHeartbeat;
+import io.kestra.jdbc.runner.JdbcQueue;
+import io.micronaut.context.ApplicationContext;
+import io.micronaut.inject.qualifiers.Qualifiers;
+import jakarta.inject.Singleton;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.function.Consumer;
+
+@Singleton
+@Slf4j
+public class JdbcWorkerJobQueueService {
+    private final JdbcQueue<WorkerJob> workerTaskQueue;
+    private final JdbcHeartbeat jdbcHeartbeat;
+    private final AbstractJdbcWorkerJobRunningRepository jdbcWorkerJobRunningRepository;
+    private final WorkerInstanceRepositoryInterface workerInstanceRepository;
+    private Runnable queueStop;
+    private WorkerInstance workerInstance;
```

Comment:
Don't store the worker instance here, better to fetch it when you need it (in delete I think)